packages/apps-config/src/endpoints/productionRelayPolkadot.ts

```diff
@@ -434,7 +433,6 @@ export const prodParasPolkadot: EndpointOption[] = [
     providers: {
       Dwellir: 'wss://eden-rpc.dwellir.com',
       OnFinality: 'wss://nodle-parachain.api.onfinality.io/public-ws',
-      Pinknode: 'wss://public-rpc.pinknode.io/nodle'
```

Comment:
CI is complaining about a dangling comma on the previous line, see https://github.com/polkadot-js/apps/actions/runs/4170604358/jobs/7219728706#step:3:316

Reply:
Resolved it.